src/bsp/src/stm32/src/interloc/src/states/src/SendFinalState.cpp

```diff
@@ -9,9 +9,9 @@ SendFinalState::SendFinalState(ILogger& logger, DecawaveArray& decawaves) :
     AbstractInterlocState(logger, decawaves) {}
 
 void SendFinalState::process(InterlocStateHandler& context) {
-    //    the delay on the send final seems to be unstable. Might need to be checked. Rarely it
-    //    causes a Tx error for no apparent reason
-    uint64_t finalTxTime = context.getTimeManager().getFinalTxTs(context.getTWR().m_pollTxTs);
+    uint64_t finalTxTime = context.getTimeManager().getFinalTxTs(
```

Comment:
I think we should maybe include the preamble in the getFinalTxTs function directly, like that there is less chances of forgetting it at some point. What do you think?

Reply:
fixed in poll, response and final transmit